packages/datagateway-download/src/ConfigProvider.tsx

```diff
@@ -23,9 +23,19 @@ export interface DownloadSettings {
   routes: PluginRoute[];
   helpSteps: { target: string; content: string }[];
   pluginHost?: string;
+
+  /**
+   * A map of UI flags that can toggle certain UI features..
+   */
+  uiFeatures: {
```

Comment:
I don't know whether it's worth making this optional & just assuming false in that case for all flags - when I was deploying I forgot to update the settings file and it thus threw an exception...

Maybe this isn't a concern though as once it's in the new settings file it's considered a "standard" config value